modules/unitbar.lua

```diff
@@ -62,22 +62,6 @@ function GladiusEx:NewUnitBarModule(name, defaults_arena, defaults_party)
 
 			self.frame[unit].statusbar:SetStatusBarColor(color.r, color.g, color.b, color.a or 1)
 			self.frame[unit].icon:SetTexture([[Interface\Glues\CharacterCreate\UI-CharacterCreate-Classes]])
-
-			-- V: wtf is "CLASS_BUTTONS"?
-			if not CLASS_BUTTONS then

```

Comment:
unitbar.lua is kind of a template for the target bar and pet bar modules. It would break the class icon in these modules.